src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/IdentityProviderFactory.php

```diff
@@ -40,6 +45,7 @@ public function __construct(array $configuration)
             Assert::keyExists($idpConfiguration, 'ssoUrl');
             Assert::keyExists($idpConfiguration, 'privateKey');
             Assert::keyExists($idpConfiguration, 'certificateFile');
+            Assert::isArray($idpConfiguration, 'attributeMapping');
```

Comment:
Maybe additionally assert that all keys and values of the array are of type string. Easily achieved using the `Assert::allString()` method.

Reply:
Done, also added missing messages to the other assertions